OpenUtau.Core/Util/LyricsHelper.cs

```diff
@@ -1,112 +1,117 @@
-﻿using System;
```

Comment:
Could you please try to make the line-ending styles match? Should be CRLF.

Reply:
I'm just about done with finals at uni so I'll take a look soon. I'm not very proficient in code so it may take me some time if I need to find someone to help.